example/lib/presentation/samples/bar/bar_chart_sample1.dart

```diff
@@ -162,7 +162,19 @@ class BarChartSample1State extends State<BarChartSample1> {
     return BarChartData(
       barTouchData: BarTouchData(
         touchTooltipData: BarTouchTooltipData(
-          tooltipBgColor: Colors.blueGrey,
+          getTooltipColor: (group) {
+            Color bgColor = switch (group.x) {
+              0 => AppColors.contentColorPurple.withAlpha(100),
```

Comment:
These colors are not consistent with the design. Please use the Colors.blueGrey just like before

Reply:
Done